exchanges/graphcache/src/store/store.ts

```diff
@@ -75,8 +75,31 @@ export class Store implements Cache {
       if (mutationType) mutationName = mutationType.name;
       if (subscriptionType) subscriptionName = subscriptionType.name;
 
-      if (this.keys) {
-        SchemaPredicates.expectValidKeyingConfig(this.schema, this.keys);
+      if (process.env.NODE_ENV !== 'production') {
+        if (this.keys) {
+          SchemaPredicates.expectValidKeyingConfig(this.schema, this.keys);
+        }
+
+        const hasUpdates =
+          Object.keys(this.updates.Mutation).length > 0 ||
```

Comment:
Nit: these will always be defined so the check here can go :)

Reply:
I'm a fan of overly-defensive checks 😊. Partly because it guards against future code changes, but also lines like `if (hasUpdates)` communicate intent and IMO help devs understand what the author was thinking.

Reply:
I’m ok with that, hence it’s just a nit 👍 Personally, I believe the types convey the intention here as our configs remain non-partial and guaranteed, so if we forgot the defaults there’d still be the tests and the types on this class that’d stand in the way of that

Reply:
Fair point. Unless anyone else has comments on this, I'll merge PR as it is?

Reply:
I think it’s fine as is 👍 just need to get the changeset fixed up and it’s ready to be merged when you’re ready 🎢